spec/mailers/user_mailer_spec.rb

```diff
@@ -48,23 +48,33 @@
   # adapted from https://github.com/spree/spree_auth_devise/blob/70737af/spec/mailers/user_mailer_spec.rb
   describe '#reset_password_instructions' do
     describe 'message contents' do
-      before do
-        @message = described_class.reset_password_instructions(user, nil)
-      end
+      let(:message) { described_class.reset_password_instructions(user, nil) }
 
       context 'subject includes' do
         it 'translated devise instructions' do
-          expect(@message.subject).to include "Reset password instructions"
+          expect(message.subject).to include "Reset password instructions"
         end
 
         it 'Spree site name' do
-          expect(@message.subject).to include Spree::Config[:site_name]
+          expect(message.subject).to include Spree::Config[:site_name]
         end
       end
 
       context 'body includes' do
         it 'password reset url' do
-          expect(@message.body.raw_source).to include spree.edit_spree_user_password_url
+          expect(message.body.raw_source).to include spree.edit_spree_user_password_url
+        end
+      end
+
+      context 'when the language is Spanish' do
```

Comment:
Because these translations will only be available once we translate it in Transifex these tests will fail. I'd rather check that we actually call `I18n.with_locale` using `expect` but also check that the subject matches `I18n.t('devise.mailer.reset_password_instructions.subject')` rather than relying on the Spanish translation. I think with these two we'll be sure the email is properly translated.

Reply:
I will do the update tonight. Thank you for your feedback @sauloperez